desktop/core/src/desktop/auth/api_authentications.py

```diff
@@ -50,15 +50,26 @@ def authenticate(self, request):
     LOG.debug('JwtAuthentication: got access token %s' % access_token)
 
     try:
-      payload = jwt.decode(access_token, 'secret', algorithms=["HS256"])
+      payload = jwt.decode(
+        access_token,
+        'secret',
+        algorithms=["RS256"],
+        verify=AUTH.VERIFY_CUSTOM_JWT.get()
+      )
     except jwt.DecodeError:
       raise exceptions.AuthenticationFailed('JwtAuthentication: Invalid token')
     except jwt.ExpiredSignatureError:
       raise exceptions.AuthenticationFailed('JwtAuthentication: Token expired')
     except Exception as e:
       raise exceptions.AuthenticationFailed(e)
+    
+    if payload.get('user') is None:
+      LOG.debug('JwtAuthentication: no user ID in token')
+      return None
+
+    LOG.debug('JwtAuthentication: got user ID %s and tenant ID %s' % (payload.get('user'), payload.get('tenantId')))
```

Comment:
@wing2fly  Added debug log for user ID and tenant ID.

Reply:
Please log the token also

Reply:
Token gets logged at Line no. 50 @wing2fly